src/main/java/cn/nukkit/command/CommandSource.java

```diff
@@ -0,0 +1,19 @@
+package cn.nukkit.command;
+
+import cn.nukkit.Server;
+import cn.nukkit.lang.TextContainer;
+import cn.nukkit.lang.TranslationContainer;
+import cn.nukkit.permission.Permissible;
+
+public interface CommandSource extends Permissible {
```

Comment:
What's the point of having both a CommandSource and CommandSender interface? Both of them do the same thing and are nearly identical.

Reply:
I’m doing it so I can work on updating the commands without having to do them all at once or there would be compile errors unless I comment everything out

Reply:
Couldn’t you just rename CommandSender to CommandSource? IntelliJ should automatically rename all references.

Reply:
True